services/web/server/tests/integration/01/test_project_delete_cleanup.py

```diff
@@ -0,0 +1,129 @@
+# pylint:disable=redefined-outer-name,unused-argument,too-many-arguments
+import logging
+from pathlib import Path
+from typing import Any, Callable, Dict, Final, Iterator, Set
+
+import aioboto3
+import aiopg
+import aiopg.sa
+import aioredis
+import pytest
+from aiohttp.test_utils import TestClient
+from models_library.projects import ProjectID
+from models_library.settings.redis import RedisConfig
+from tenacity._asyncio import AsyncRetrying
+from tenacity.before_sleep import before_sleep_log
+from tenacity.stop import stop_after_attempt
+from tenacity.wait import wait_fixed
+from yarl import URL
+
+log = logging.getLogger(__name__)
+
+pytest_simcore_core_services_selection = [
+    "catalog",
+    "dask-scheduler",
+    "director-v2",
+    "director",
+    "migration",
+    "postgres",
+    "rabbit",
+    "redis",
+    "storage",
+]
+pytest_simcore_ops_services_selection = ["minio", "adminer"]
+
+S3_DATA_REMOVAL_SECONDS: Final[int] = 2
+API_PREFIX = "/v0"
+
+# FIXTURES
+
+
+@pytest.fixture(autouse=True)
+def __drop_and_recreate_postgres__(
+    database_from_template_before_each_function,
+) -> Iterator[None]:
+    yield
+
+
+@pytest.fixture(autouse=True)
+async def __delete_all_redis_keys__(redis_service: RedisConfig):
+    client = await aioredis.create_redis_pool(redis_service.dsn, encoding="utf-8")
+    await client.flushall()
+    client.close()
+    await client.wait_closed()
+
+    yield
+    # do nothing on teadown
+
+
+# UTILS
+
+
+async def _fetch_stored_files(
+    minio_config: Dict[str, Any], project_id: ProjectID
+) -> Set[str]:
+
+    s3_config: Dict[str, str] = minio_config["client"]
+
+    def _endpoint_url() -> str:
+        protocol = "https" if s3_config["secure"] else "http"
```

Comment:
good to know, so we shall need to search for all the endpoint functions and refactor when porting to the new settings

Reply:
This one is a bit of a mehh thing. Since we do not use S3 in the webserver we do not have configurations for it captured on other settings. That's why it looks so wired.